deploy/charts/cert-manager/templates/deployment.yaml

```diff
@@ -54,8 +54,8 @@ spec:
       priorityClassName: {{ .Values.global.priorityClassName | quote }}
       {{- end }}
       {{- $enabledDefined := gt (len (keys (pick .Values.securityContext "enabled"))) 0 }}
-      {{- $legacyEnabledExplicitlydOff := and $enabledDefined (not .Values.securityContext.enabled) }}
-      {{- if and .Values.securityContext (not $legacyEnabledExplicitlydOff) }}
+      {{- $legacyEnabledExplicitlyOff := and $enabledDefined (not .Values.securityContext.enabled) }}
+      {{- if and .Values.securityContext (not $legacyEnabledExplicitlyOff) }}
```

Comment:
It's possible this is an API change

Reply:
Nope this should be okay! 😄 it's a variable definition and not exposed to users.